### ISSUE TITLE
<b-input-group-text> does not render

### ISSUE THREAD

**Issue by luckylooke** [bootstrap-vue/bootstrap-vue#3381]
### Describe the bug
<b-input-group-text> does not render, even on docs site. https://bootstrap-vue.js.org/docs/components/input-group, see second input should have "!" appended, but shows nothing.

Do not have much time now to examine more, sorry.

### Expected behavior

"!" to be appended into input field

### Versions

**Libraries:**
latest, see docs

**Environment:**

- Device: Mac
- OS: High Sierra
- Browser: Chrome
- Version: 74.0.3729.157

### Demo link

https://bootstrap-vue.js.org/docs/components/input-group

I will bypass it with other possibilities of b-input for now.
Thank you

Happy coding :)

**Comment by luckylooke** [comment]
Additional info: "prepend" does work as expected.

**Comment by luckylooke** [comment]
Another finding:
Using dropdown as prepend component make even append group text work.. removing dropdown breaks it again.
```
<b-input-group>
    <b-input-group-text slot="append">Username</b-input-group-text>
    <b-form-input></b-form-input>

    <b-dropdown text="Dropdown" variant="success" slot="prepend">
      <b-dropdown-item>Action A</b-dropdown-item>
      <b-dropdown-item>Action B</b-dropdown-item>
    </b-dropdown>
  </b-input-group>
```

**Comment by jackmu95** [comment]
This has already been fixed by #3321 and will be coming with RC21.

**Comment by tmorehouse** [comment]
Version 2.0.0-rc.21 has been released
